catalog/spatial/spatial-app/src/main/resources/features.xml

```diff
@@ -133,9 +133,9 @@
     <feature name="webservice-gazetteer" version="${project.version}"
              description="Gazetteer service utilizing the Geonames.org web service.">
         <feature>spatial-geocoding-endpoint</feature>
-        <bundle>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>
-        <bundle>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>
-        <bundle>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>
+        <bundle>mvn:com.fasterxml.jackson.core/jackson-core/${jackson.version}</bundle>
+        <bundle>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson.version}</bundle>
+        <bundle>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson.version}</bundle>
```

Comment:
✏️ Start the jackson feature defined in catalog-app or security-app, instead.
https://github.com/codice/ddf/blob/0af70deb885c1b8734fcc90fab21a8421996ebf3/features/security/src/main/feature/feature.xml#L405-L409
```xml
<feature name="webservice-gazetteer" version="${project.version}"
         description="Gazetteer service utilizing the Geonames.org web service.">
    <feature>spatial-geocoding-endpoint</feature>
    <feature>jackson</feature>
    <bundle>mvn:org.codice.ddf.spatial/spatial-geocoding-websearch/${project.version}</bundle>
    <bundle>mvn:org.codice.ddf.spatial/spatial-geocoding-plugin/${project.version}</bundle>
</feature>
```

Reply:
Done